doc/dune-files.rst

```diff
@@ -629,6 +629,25 @@ the description of an opam switch, as follows:
   execution counters is *src/fdo.exe.fdo-profile*.  This feature is
   **experimental** and no backwards compatibility is implied.
 
+- ``(watch_exclusions <regex1> ... <regexN>)`` configures a set of regular
```

Comment:
It would be nice to be able to use the ordered set language here https://dune.readthedocs.io/en/stable/concepts.html#ordered-set-language. The standard set of exclusions would be `:standard` and one would be able to replace that if one wanted.

Currently `Ordered_set_lang` lives in `dune_rules`, may have to be moved to `dune_engine` for this to work, I think.

It would also allow things like `(:standard \ _opam)` which would be useful in your (@jonahbeckford's) particular case, no?